lua/starfall/libs_cl/vgui.lua

```diff
@@ -1012,7 +1030,7 @@ end
 --- Set a function to run when a file is selected.
 --@param function callback Function to run. Has 1 argument which is the filepath to the selected file.
 function dfb_methods:onSelect(func)
-	function unwrap(self):OnSelect(filepath, selpnl)
+	unwrap(self).OnSelect = function(filepath, selpnl)

```

Comment:
Functions args might need to be updated since you changed the colon to a dot.

Reply:
It kept erroring with the colons (unexpected symbol or w/ever), which is why I changed it. I haven't tested the panels like the filebrowser specifically, but all my chips using VGUI seem to work fine.

I'll give them a test anyway but I think they should be fine.

Reply:
`obj:method = function(a, b) end` is a syntax error.
`function obj:method(a, b)` is the correct format and is syntax sugar for `function obj.method(self, a, b)` meant to be called with `obj:method(a, b)` which is basically the same as `obj.method(obj, a, b)`

Reply:
This bug still exists. Please test/fix it.

Reply:
Fixed.